tests/console/validate_partitioning.pm

```diff
@@ -0,0 +1,100 @@
+# Suse'S openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple LVM partition validation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use y2logsstep;
+
+sub run {
+    my $self     = shift;
+    my $expected_lv_stats = {
+        write_access => qr/\s{2}LV Write Access \s+ read\/write/,
```

Comment:
` \s+ ` means 3 spaces minimum, perhaps just `\s+`?
Perhaps not being so specific with `\s+` instead of \s{2} (not sure if could potentially change to a tab in the future). Up to you both, it is just for maintenance of the test and really minor things.